elide-annotations/src/main/java/com/yahoo/elide/security/checks/prefab/Role.java

```diff
@@ -31,4 +31,19 @@ public boolean ok(User user) {
             return false;
         }
     }
+
+    /**
+     * Check which verifies if the user is a member of a particular role.
+     */
```

Comment:
How (if at all) will the Role class here relate to https://docs.spring.io/spring-security/site/docs/4.2.12.RELEASE/apidocs/org/springframework/security/core/GrantedAuthority.html in https://docs.spring.io/spring-security/site/docs/4.2.12.RELEASE/apidocs/org/springframework/security/core/Authentication.html for the spring security use case?

Reply:
GrantedAuthority can be mapped to a string.  You would pass in the string representation of the GrantedAuthority in that case.